source/rust_verify_test/tests/recursion.rs

```diff
@@ -103,13 +103,43 @@ test_verify_one_file! {
         }
 
         proof fn count_down_properties() {
+            assert(count_down_a(1) == 1);
+        }
+
+        proof fn count_down_properties_fail() {
+            assert(count_down_a(2) == 2);   // FAILS
+        }
+    } => Err(err) => assert_one_fails(err)
+}
+
+test_verify_one_file! {
+    // Test that default fuel only provides one cycle of unfolding
+    #[test] mutually_recursive_expressions_insufficient_fuel_short_cycle verus_code! {
+        spec fn count_down_a(i: nat) -> nat
```

Comment:
Would it be worth adding an explicit comment here about the fact that `count_down_a` now calls itself, meaning the shortest cycle is 1, which is why the assertion that `count_down_a(1) == 1` fails without more fuel?